soc/Makefile

```diff
@@ -150,25 +162,30 @@ FLASH_FREQ=38.8 #MHz
 all: $(PROJ).svf
 
 $(PROJ).json $(PROJ).blif: $(SRC) $(SRC_SYNTH) $(EXTRA_DEPEND)
-	yosys$(EXE) -e ".*(assigned|implicitly).*" -l yosys.log \
+	yosys -e ".*(assigned|implicitly).*" -l yosys.log \
 		-p "read -sv -DBADGE_VER=$(BADGE_VER) -DBADGE_V$(BADGE_VER)=1 $(SRC) $(SRC_SYNTH); \
 			  synth_ecp5 -abc9 -top top_fpga -json $(PROJ).json -blif $(PROJ).blif"
 
 %_out_synth.config: %.json clock-constrainsts.py
-	nextpnr-ecp5$(EXE) --json $< --lpf $(CONSTR) --textcfg $@ --45k --package CABGA381 --speed 8 \
+	nextpnr-ecp5 --json $< --lpf $(CONSTR) --textcfg $@ --45k --package CABGA381 --speed 8 \
 			--pre-pack clock-constrainsts.py -l nextpnr.log --freq 48 --seed $(SEED)
 
-%_out.config: %_out_synth.config rom.hex
-	ecpbram -i $< -o $@ -f rom_random_seeds0x123456.hex -t rom.hex
+%_out.config: %_out_synth.config bram_replace/bram_replace$(EXE) rom.hex
+	bram_replace/bram_replace$(EXE) -l 8192 -r $< $@ rom.hex
+
+bram_replace/bram_replace:
```

Comment:
Something went wrong for sure here; this was removed from master a while ago and your PR brings it back from the dead.

Reply:
Yup.  I did some sketchy rebasing along the way, and it bit me.  Will do a clean PR.